api/v1alpha1/workspace_condition_types.go

```diff
@@ -13,8 +13,11 @@ const (
 	// WorkspaceConditionTypeResourceProvisioned is the state when Resources have been created.
 	WorkspaceConditionTypeResourceProvisioned = ConditionType("ResourceProvisioned")
 
-	// WorkspaceConditionTypeConfigured is the state when Resources have been Configured.
-	WorkspaceConditionTypeConfigured = ConditionType("ResourceConfigured")
+	// WorkspaceConditionTypeMachineStatus is the state when checking machine status.
+	WorkspaceConditionTypeMachineStatus = ConditionType("MachineStatus")
+
+	// WorkspaceConditionTypeInstallNodePlugins is the state when installing node plugins.
+	WorkspaceConditionTypeInstallNodePlugins = ConditionType("InstallNodePlugins")
```

Comment:
Let us be consistent for the status:
MachineStatus -> MachineIsReady
InstallNodePlugins -> NodePluginsInstalled

Reply:
Addressed.